configurations/manager/large_dataset.yaml

```diff
@@ -0,0 +1,5 @@
+test_duration: 1200
+
+instance_type_db: 'i3.xlarge'
+
+stress_cmd: "cassandra-stress write cl=QUORUM n=200200300 -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=2,us-west-2scylla_node_west=1)' -port jmx=6868 -mode cql3 native -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=400200300..600400600"
```

Comment:
why "400200300" and not just zero?

Reply:
I'm pretty sure I copied the command from another test scenario. Didn't give it too much thought, does it matter?